autoPyTorch/pipeline/components/setup/early_preprocessor/EarlyPreprocessing.py

```diff
@@ -67,8 +67,8 @@ def get_hyperparameter_search_space(
     @staticmethod
     def get_properties(dataset_properties: Optional[Dict[str, Any]] = None) -> Dict[str, str]:
         return {
-            'shortname': 'Preprocessing',
-            'name': 'Preprocessing Node',
+            'shortname': 'EarlyPreprocessing',
```

Comment:
haha my PR and your PR will class on this change. Depending on which get merged first, this will have to be updated